front/form_language.form.php

```diff
@@ -31,7 +31,7 @@
 
 include ('../../../inc/includes.php');
 
-Session::checkRight('entity', UPDATE);
+Session::checkRight(PluginFormcreatorForm::$rightname, UPDATE);
```

Comment:
I think this would add unnecessary complexity.
If the profile only has the right to update and not the right to create, does that mean that it can modify translations, but not add them?